src/getCoinCombination.test.js

```diff
@@ -7,4 +7,40 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should handle 0', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('should return pennies', () => {
+    expect(getCoinCombination(3)).toEqual([3, 0, 0, 0]);
+  });
+
+  it('should return nickels', () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it('should return dimes', () => {
+    expect(getCoinCombination(20)).toEqual([0, 0, 2, 0]);
+  });
+
+  it('should return quarters', () => {
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
+  });
+
+  it('should count correctly up to 100', () => {
+    expect(getCoinCombination(88)).toEqual([3, 0, 1, 3]);
+  });
+
+  it('should count correctly up to 1000', () => {
+    expect(getCoinCombination(982)).toEqual([2, 1, 0, 39]);
+  });
+
+  it('should handle negative', () => {
+    expect(getCoinCombination(-4)).toEqual([0, 0, 0, 0]);
```

Comment:
The test case for negative values expects the function to return `[0, 0, 0, 0]`. This behavior should be explicitly defined in the implementation. Consider whether returning an empty array or throwing an error might be more appropriate, depending on the requirements.

Reply:
changed to an error